corehq/apps/es/domains.py

```diff
@@ -78,12 +78,12 @@ def _from_dict(self, domain_dict):
 
         sub = Subscription.visible_objects.filter(subscriber__domain=domain_dict['name'], is_active=True)
         domain_dict['deployment'] = domain_dict.get('deployment') or {}
-        countries = domain_dict['deployment'].get('countries', [])
+        domain_countries = domain_dict['deployment'].get('countries', [])
         domain_dict['deployment']['countries'] = []
         if sub:
             domain_dict['subscription'] = sub[0].plan_version.plan.edition
-        for country in countries:
-            domain_dict['deployment']['countries'].append(Countries[country].upper())
+        for country in domain_countries:
+            domain_dict['deployment']['countries'].append(dict(countries)[country].upper())
```

Comment:
This could be made a tiny bit more efficient by building the dict once outside the loop.
```suggestion
        countries_map = dict(countries)
        for country in domain_countries:
            domain_dict['deployment']['countries'].append(countries_map[country].upper())
```